math-libs/BLAS/CMakeLists.txt

```diff
@@ -264,6 +267,10 @@ therock_cmake_subproject_declare(hipBLAS
     -DROCM_PATH=
     -DROCM_DIR=
     "-DBUILD_WITH_SOLVER=${THEROCK_ENABLE_SOLVER}"
+    -DBUILD_CLIENTS_TESTS=${THEROCK_BUILD_TESTING}
+    -DBUILD_CLIENTS_BENCHMARKS=${THEROCK_BUILD_TESTING}
```

Comment:
style is inconsistent here, add quotes to the new lines or remove from the prior line?

Reply:
Yes, noticed that and planned to send a follow which fixes some others style inconsistencies as well which are unrelated to this change.

Reply:
Okay, as long as it doesn't sit too long. Project-wide cleanup SGTM.